src/loader.ts

```diff
@@ -28,6 +28,11 @@ export interface Has {
 	add(name: string, value: any, now?: boolean, force?: boolean): void;
 }
 
+export interface LoaderError extends Error {
+	src: string;
+	info: any;
```

Comment:
It's an object, yes, but you're passing specific things with that object. From the call later in the file:

```ts
							makeSignalError(message, {
								module,
								url,
								parentMid
							});
```

Will the info be different for different errors? Will there always be some properties that will be there?